addons/message_try_firefox_extension/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "api_version": "0.1",
+  "id": "message_try_firefox_extension",
+  "name": "Try the Firefox extension",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "3.25.1",
```

Comment:
So that QA can start testing this we could do `"env": "staging",` instead.

Reply:
Thanks, I thought I was forgetting one. I've updated this, and will update that follow up ticket in a moment.